docs/Reference/Certificates/remoteProvisioning.md

```diff
@@ -0,0 +1,90 @@
+| 4. Using the rpc-go client application to activate the remote device | Deploy and run the client as Administrator on the desired remote platform. Leverage the profile you created in Step 2.  |
+
+
+Figure 1 illustrates the steps below. Also, see Get Started
+
+To manage a device remotely, you must *provision* it, securely configure it to communicate with the development system, the provisioning server. The toolkit accomplishes secure provisioning with a process of authentication and a tunnel using Transport Layer Security (TLS) protocol. 
+
+This authentication process involves:
+
+* **A provisioning certificate*:** an electronic document used to establish the identity of the holder
+* **CIRA:** a protocol used by the remote device to "call home", initiate remote connectivity, with a remote provisioning server.
+
+After a device is provisioned, the administrator can use the development system to perform different control activities, such as [power actions (e.g., Reset)](../powerstates.md) without an onsite visit.
+
+<figure class="figure-image">
+<img src="..\..\..\assets\images\RemoteProvCert.png" alt="Figure 1: Certificates in the Toolkit: Provisioning and CIRA">
+<figcaption>Figure 1: Provisioning and CIRA</figcaption>
+</figure>
+
+## What about manual provisioning?
+
+Sometimes it's not possible to deploy the rpc-go client application to activate the device remotely. When remote provisioning is not an option, the provisioning process may involve physically touching the devices. 
+
+To provision manually for ACM, see [Set a DNS Suffix through MEBX](../MEBX/dnsSuffix.md).
+
+## What is a remote provisioning certificate?
+
+A remote provisioning certificate is used for the activation of the remote device by the provisioning server, the development system. The activation phase of provisioning *turns on* or enables Intel AMT in the remote device. The Intel AMT firmware contains the root certificate hashes, TLS-based thumbprints, of participating vendors, known as certificate authorities (CA). The root certificate hash is compared with that certificate's hash. If they match, the certificate is deemed valid. 
+
+!!! Note
+    * The remote provisioning certificate is used only for initial provisioning of the device, bringing it under the management of the provisioning server. 
+    * Purchase a separate certificate for website hosting. 
+
+## How do provisioning certificates work in the Toolkit?
+
+### Purchase
+
+To use a provisioning certificate in the toolkit, purchase the certificate from a CA. 
+
+!!! Note
+    * You do not have to purchase a certificate for each device in a fleet. The certificate is purchased for the provisioning server. 
+    * Prices vary per vendor.
+
+To get a list of participating vendors, see [Intel Developer Zone](https://www.intel.com/content/www/us/en/developer/topic-technology/edge-5g/tools/open-amt-cloud-toolkit.html).
```

Comment:
We can jump them right to it by appending #linklistblade-3

i.e. https://www.intel.com/content/www/us/en/developer/topic-technology/edge-5g/tools/open-amt-cloud-toolkit.html#linklistblade-3